lib/AST/GenericSignatureBuilder.cpp

```diff
@@ -222,7 +226,7 @@ bool RequirementSource::isInferredRequirement() const {
 }
 
 unsigned RequirementSource::classifyDiagKind() const {
```

Comment:
Unrelated to these changes, but is there a reason this is `unsigned` instead of an enum?